File: Atividade_1/CalculoDeImpostoJava/src/CalculadoraImposto.java
public class CalculadoraImposto {

    private double preco;
    private boolean ehAlimento;

    public CalculadoraImposto() {
    }

    public double calculadoraImposto(boolean ehAlimento, double preco) throws PrecoInvalidoException {
        if (preco < 0){
            throw new PrecoInvalidoException("O preço não pode ser negativo.");
        }
        if (ehAlimento){
            return preco * 0.05;
        }else {
            return preco * 0.10;
        }
    }
}

File: Atividade_1/CalculoImpostoPy/CalculadoraImposto.py
from PrecoInvalidoException import PrecoInvalidoException

class CalculadoraImposto:
    def __init__(self):
        pass 

    def calcular_imposto(self, ehAlimento, preco):
        if preco < 0:
            raise PrecoInvalidoException("O preço não pode ser negativo.")
        
        if(ehAlimento):
            return preco * 0.05
        else:
            return preco * 0.10
